sda/internal/broker/broker_test.go

```diff
@@ -32,15 +32,9 @@ var tMqconf = MQConf{}
 
 func TestMain(m *testing.M) {
 	certPath, _ = os.MkdirTemp("", "gocerts")
-	defer os.RemoveAll(certPath)
```

Comment:
Although I assume you have good reason, I don't really see why this change is done..? What if there is panic somewhere before `m.Run()`, would line 106 really be reached?

Reply:
The linter complained about not having an `os.Exit` and then it started to complain about the `defer` not being called due to the `os.Exit`.

The other option is of course to just ignore the linter complaints about not having an `os.Exit`

Reply:
Hehe, interesting. I think ignore the linter for that line sounds like a nice(r) option then, so that we don't end up with unwanted files/dirs hanging around. What do you think?

Reply:
I think that since the cert files are created in the system's temp dir, they will be cleaned up at some point either way so this shouldn't be a problem.

Reply:
Ok, good point. I resolve this comment and approve. If the `defer` is re-added and linter-ignored, I will approve that as well :smile: